docs/solutions/Hadoop/EL7/Cloudera/Development/scripts/bastion.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/bash
+## Bastion script to drive all provisioning tasks 
+## by Zachary Smith (Zachary.Smith@oracle.com) 
```

Comment:
The "by" and "Last Update" lines are not a pattern that we've typically followed.

Reply:
Removing these.